setup/install_upstream_keylime/test.sh

```diff
@@ -32,20 +32,21 @@ gpgkey=https://download.copr.fedorainfracloud.org/results/scorreia/keylime/pubke
 repo_gpgcheck=0
 enabled=1
 enabled_metadata=1
+priority=999
 _EOF'
-            RHEL_EXTRA_PKGS="cfssl python3-pip"
             if rlIsRHEL 8 || rlIsCentOS 8; then
+                RHEL_EXTRA_PKGS="cfssl python3-pip"
                 RHEL_EXTRA_PIP_PKGS="packaging"
             else
-                RHEL_EXTRA_PKGS="$RHEL_EXTRA_PKGS python3-packaging"
+                RHEL_EXTRA_PKGS="cfssl python3-packaging python3-lark-parser"
```

Comment:
We removed support for `cfssl` a while ago, so this can go away as well

Reply:
I thought it is necessary for Python agent. If not, I will remove it.

Reply:
It's not. Here's the PR that removed support for it: https://github.com/keylime/keylime/pull/1012